op-node/rollup/superchain.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/ethereum-optimism/superchain-registry/superchain"
 )
 
-var OPStackSupport = params.ProtocolVersionV0{Build: [8]byte{}, Major: 4, Minor: 0, Patch: 0, PreRelease: 1}.Encode()
+var OPStackSupport = params.ProtocolVersionV0{Build: [8]byte{}, Major: 5, Minor: 0, Patch: 0, PreRelease: 1}.Encode()
```

Comment:
We're gonna run into the issue where we may never cut a version at v4.0.0

Reply:
The version is meant to indicate the protocol it supports up to. There doesn't strictly need to be a whole release in this case. 5.0.0-1 implies 4.x.x support, and Canyon support is stable